### PR TITLE
Added Objects, partial completion with no documentation

### PR DESCRIPTION
**Pull request by garyjlin**

### CODE REVIEW

File: Objects/Main.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package Objects;
+import Objects.*;
+
+public class Main {
```
Comment (YuFuTotG) [inline Objects/Main.java:4]: Why is there a Main.java
I don't understand the purpose of this. is it necessary?

**Comment by YuFuTotG** [comment]
There is only one commit here. Please commit more frequently.
I have made a note above, please comment on it when you can.

**Comment by YuFuTotG** [comment]
Also please include a description.

File: Objects/Main.java (hunk 1 above)
Comment (garyjlin) [inline Objects/Main.java:4]: Nah that was for Objects testing, I've removed it in the latest commit.

**Pull request merged**